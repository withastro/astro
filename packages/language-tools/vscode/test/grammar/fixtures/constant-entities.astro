&#x210A; &#8458; &gscr; &#x210B; &#8459; &hamilt;

<div>
  &#x210A; &#8458; &gscr; &#x210B; &#8459; &hamilt;<div>
    &#x210A; &#8458; &gscr; &#x210B; &#8459; &hamilt;
  </div>
</div>
