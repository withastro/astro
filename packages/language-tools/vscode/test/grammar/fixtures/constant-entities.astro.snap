>&#x210A; &#8458; &gscr; &#x210B; &#8459; &hamilt;
#^ source.astro constant.character.entity.numeric.hexadecimal.astro punctuation.definition.entity.astro
# ^^^^^^^ source.astro constant.character.entity.numeric.hexadecimal.astro
#        ^ source.astro
#         ^ source.astro constant.character.entity.numeric.decimal.astro punctuation.definition.entity.astro
#          ^^^^^^ source.astro constant.character.entity.numeric.decimal.astro
#                ^ source.astro
#                 ^ source.astro constant.character.entity.named.gscr.astro punctuation.definition.entity.astro
#                  ^^^^ source.astro constant.character.entity.named.gscr.astro
#                      ^ source.astro constant.character.entity.named.gscr.astro punctuation.definition.entity.astro
#                       ^ source.astro
#                        ^ source.astro constant.character.entity.numeric.hexadecimal.astro punctuation.definition.entity.astro
#                         ^^^^^^^ source.astro constant.character.entity.numeric.hexadecimal.astro
#                                ^ source.astro
#                                 ^ source.astro constant.character.entity.numeric.decimal.astro punctuation.definition.entity.astro
#                                  ^^^^^^ source.astro constant.character.entity.numeric.decimal.astro
#                                        ^ source.astro
#                                         ^ source.astro constant.character.entity.named.hamilt.astro punctuation.definition.entity.astro
#                                          ^^^^^^ source.astro constant.character.entity.named.hamilt.astro
#                                                ^ source.astro constant.character.entity.named.hamilt.astro punctuation.definition.entity.astro
>
><div>
#^ source.astro meta.scope.tag.div.astro meta.tag.start.astro punctuation.definition.tag.begin.astro
# ^^^ source.astro meta.scope.tag.div.astro meta.tag.start.astro entity.name.tag.astro
#    ^ source.astro meta.scope.tag.div.astro meta.tag.start.astro punctuation.definition.tag.end.astro
>  &#x210A; &#8458; &gscr; &#x210B; &#8459; &hamilt;<div>
#^^ source.astro text.astro
#  ^ source.astro text.astro constant.character.entity.numeric.hexadecimal.astro punctuation.definition.entity.astro
#   ^^^^^^^ source.astro text.astro constant.character.entity.numeric.hexadecimal.astro
#          ^ source.astro text.astro
#           ^ source.astro text.astro constant.character.entity.numeric.decimal.astro punctuation.definition.entity.astro
#            ^^^^^^ source.astro text.astro constant.character.entity.numeric.decimal.astro
#                  ^ source.astro text.astro
#                   ^ source.astro text.astro constant.character.entity.named.gscr.astro punctuation.definition.entity.astro
#                    ^^^^ source.astro text.astro constant.character.entity.named.gscr.astro
#                        ^ source.astro text.astro constant.character.entity.named.gscr.astro punctuation.definition.entity.astro
#                         ^ source.astro text.astro
#                          ^ source.astro text.astro constant.character.entity.numeric.hexadecimal.astro punctuation.definition.entity.astro
#                           ^^^^^^^ source.astro text.astro constant.character.entity.numeric.hexadecimal.astro
#                                  ^ source.astro text.astro
#                                   ^ source.astro text.astro constant.character.entity.numeric.decimal.astro punctuation.definition.entity.astro
#                                    ^^^^^^ source.astro text.astro constant.character.entity.numeric.decimal.astro
#                                          ^ source.astro text.astro
#                                           ^ source.astro text.astro constant.character.entity.named.hamilt.astro punctuation.definition.entity.astro
#                                            ^^^^^^ source.astro text.astro constant.character.entity.named.hamilt.astro
#                                                  ^ source.astro text.astro constant.character.entity.named.hamilt.astro punctuation.definition.entity.astro
#                                                   ^ source.astro meta.scope.tag.div.astro meta.tag.start.astro punctuation.definition.tag.begin.astro
#                                                    ^^^ source.astro meta.scope.tag.div.astro meta.tag.start.astro entity.name.tag.astro
#                                                       ^ source.astro meta.scope.tag.div.astro meta.tag.start.astro punctuation.definition.tag.end.astro
>    &#x210A; &#8458; &gscr; &#x210B; &#8459; &hamilt;
#^^^^ source.astro text.astro
#    ^ source.astro text.astro constant.character.entity.numeric.hexadecimal.astro punctuation.definition.entity.astro
#     ^^^^^^^ source.astro text.astro constant.character.entity.numeric.hexadecimal.astro
#            ^ source.astro text.astro
#             ^ source.astro text.astro constant.character.entity.numeric.decimal.astro punctuation.definition.entity.astro
#              ^^^^^^ source.astro text.astro constant.character.entity.numeric.decimal.astro
#                    ^ source.astro text.astro
#                     ^ source.astro text.astro constant.character.entity.named.gscr.astro punctuation.definition.entity.astro
#                      ^^^^ source.astro text.astro constant.character.entity.named.gscr.astro
#                          ^ source.astro text.astro constant.character.entity.named.gscr.astro punctuation.definition.entity.astro
#                           ^ source.astro text.astro
#                            ^ source.astro text.astro constant.character.entity.numeric.hexadecimal.astro punctuation.definition.entity.astro
#                             ^^^^^^^ source.astro text.astro constant.character.entity.numeric.hexadecimal.astro
#                                    ^ source.astro text.astro
#                                     ^ source.astro text.astro constant.character.entity.numeric.decimal.astro punctuation.definition.entity.astro
#                                      ^^^^^^ source.astro text.astro constant.character.entity.numeric.decimal.astro
#                                            ^ source.astro text.astro
#                                             ^ source.astro text.astro constant.character.entity.named.hamilt.astro punctuation.definition.entity.astro
#                                              ^^^^^^ source.astro text.astro constant.character.entity.named.hamilt.astro
#                                                    ^ source.astro text.astro constant.character.entity.named.hamilt.astro punctuation.definition.entity.astro
>  </div>
#^^ source.astro text.astro
#  ^^ source.astro meta.scope.tag.div.astro meta.tag.end.astro punctuation.definition.tag.begin.astro
#    ^^^ source.astro meta.scope.tag.div.astro meta.tag.end.astro entity.name.tag.astro
#       ^ source.astro meta.scope.tag.div.astro meta.tag.end.astro punctuation.definition.tag.end.astro
></div>
#^^ source.astro meta.scope.tag.div.astro meta.tag.end.astro punctuation.definition.tag.begin.astro
#  ^^^ source.astro meta.scope.tag.div.astro meta.tag.end.astro entity.name.tag.astro
#     ^ source.astro meta.scope.tag.div.astro meta.tag.end.astro punctuation.definition.tag.end.astro
>